andela/baby-face-rc
# #154872209 Integrate Paystack payment gateway

#### What does this PR do?
Integrate Paystack payment gateway
#### Description of Task to be completed?
Make  Paystack payment available as one of the payment options
#### How should this be manually tested?
Add a product to cart
Enter your shipping address
Select your shipping preference
Select Paystack Payment and Complete your Order
A Paystack payment gateway pop up would appear
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#154872209
#### Screenshots (if appropriate)
<img width="1439" alt="screen shot 2018-02-09 at 10 33 29 am" src="https://user-images.githubusercontent.com/32909231/36021203-587d50a8-0d85-11e8-93dd-94dd9f98696a.png">

#### Questions:
N/A
